android/app/src/main/java/app/covidshield/utils/PendingTokenManager.kt

```diff
@@ -0,0 +1,44 @@
+package app.covidshield.utils
+
+import android.annotation.SuppressLint
+import android.content.Context
+import app.covidshield.MainApplication
+
+private const val TOKEN = "TOKEN"
```

Comment:
Should get set via an .env var?

Reply:
This is just RANDOM string. No need to use .env.